core/tests/style/pack/test_deprecated_properties.py

```diff
@@ -31,9 +31,7 @@ def delitem(obj, name):
 @pytest.mark.parametrize(
     "old_name, new_name, value, default",
     [
-        # Travertino 0.3.0 doesn't support accessing a directional property via bracket
-        # notation.
-        # ("padding", "margin", (5, 5, 5, 5), (0, 0, 0, 0)),
+        ("padding", "margin", (5, 5, 5, 5), (0, 0, 0, 0)),
```

Comment:
Wow, it hadn't even occurred to me that I'd made this work. Nice catch! 🤣

Reply:
Actually wait, my attempt to deduplicate logic (by having `__getitem__` call `__getattribute__`) had the happy accident of letting bracket notation access `padding`, but only because it entirely bypasses `BaseStyle.__getitem__`, which verifies valid property names. Now you can set `style['bogus-name']` and it won't complain.

This mechanism is currently enforced in Travertino's tests, but not Toga's, so I missed it. I'll make a fix and add a test.